src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -459,7 +459,7 @@ private WindowTransparencyLevel Win10EnableBlur(WindowTransparencyLevel transpar
             }
         }
 
-        public IEnumerable<object> Surfaces => new object[] { Handle, _gl, _framebuffer };
+        public IEnumerable<object> Surfaces => new object[] { Handle, _gl, _framebuffer};
```

Comment:
WindowImplPlatformHandle is not actually exposed here

Reply:
Like this, `public IEnumerable<object> Surfaces => new object[] { (IPlatformNativeSurfaceHandle)Handle, _gl, _framebuffer };`.
The Handle is still an object of type `IPlatformNativeSurfaceHandle`.